demo/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion "Google Inc.:Google APIs:21"
```

Comment:
I assume that's possible, although I've never configured that setup before.  Are you able to see if anyone on the Android team can explain why the API 23 Google API emulator isn't working with gradle commands?

http://stackoverflow.com/questions/32952413/gradle-commands-fail-on-api-23-google-api-emulator-image-armeabi-v7a

If the emulator issue isn't something that's going to be fixed anytime soon I can look into running on API 21 only on Travis.